dash/backend/src/app.module.ts

```diff
@@ -141,9 +141,8 @@ const myFormatter = info => {
         enabled: true
       },
     }),
-    ScheduleModule.forRoot(),
```

Comment:
Wait, did we intend to remove this? I think another change Kyle worked on might have required this.

Reply:
Yes. It was only added to account for the old metrics cron job.